### ISSUE TITLE
pip10 breaks pybind11

### ISSUE THREAD

**Issue by ddu1** [facebookresearch/fastText#493]
After upgrading to pip10, the python build will throw the following error:

`python/fastText/pybind/fasttext_pybind.cc:13:31: fatal error: pybind11/pybind11.h: No such file or directory`

pip can no longer find the includes of pybind11. 
The issue is really a pybind11 issue. See the discussion at https://github.com/pybind/pybind11/issues/1174

**Comment by EdouardGrave** [comment]
Hi @ddu1,

Thank you for reporting this issue.

Does using the latest version of pybind11 fixes the issue (c.f. #488)?

Best,
Edouard

**Comment by ddu1** [comment]
@EdouardGrave,
Yes, as mentioned in #488, the fix 060936f of pybind11 solves the issue.
However before the next release of pybind11, I would simply change `setup.py` a bit, 

`install_requires=["pybind11>=2.2", "setuptools >= 0.7.0", "numpy", "pip>=9,<10" ]`
